### PR TITLE
LPS-166547 Uses new taglib

### PR DESCRIPTION
**Pull request by ealonso**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by ealonso** [comment]
ci:test:sf

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:test:relevant

File: modules/apps/frontend-taglib/frontend-taglib/src/main/java/com/liferay/frontend/taglib/servlet/taglib/IconOptionsTag.java [hunk 1]
```diff
@@ -133,7 +133,8 @@ else if (Validator.isNotNull(url)) {
 							}
 							else {
 								dropdownItem.setData(
-									portletConfigurationIcon.getData());
+									portletConfigurationIcon.getContenxt(
```
Comment (ethib137) [inline modules/apps/frontend-taglib/frontend-taglib/src/main/java/com/liferay/frontend/taglib/servlet/taglib/IconOptionsTag.java:136]: @ealonso looks like a small typo: getContenxt -> getContext

**Comment by ozysouza** [comment]
Just started reviewing :)

**Comment by ozysouza** [comment]
Please see https://github.com/liferay-echo/liferay-portal/pull/10112

**Pull request closed without merging**